README.md

```diff
@@ -1,21 +1,27 @@
 # Swift for Visual Studio Code
 
-## Installation
+This extension adds language support for Swift to Visual Studio Code. It supports:
+
+* Automatic task creation
+* Package dependency view
+* Code completion
+* Jump to definition, peek definition, find all references, symbol search
+* Error annotations and apply suggestions from errors
+* Automatic generation of launch configurations for debugging with [CoreLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
+
+Swift support uses [SourceKit LSP](https://github.com/apple/sourcekit-lsp) for the [language server](https://microsoft.github.io/language-server-protocol/overviews/lsp/overview/) to power code completion and [LLDB](https://github.com/vadimcn/vscode-lldb) to enable debugging. 
 
-Currently, the only way to install this extension is to build it from source. To do this, you'll need to have [Node.js](https://nodejs.org) installed. On Linux, make sure to install Node.js from its official website or from [NodeSource](https://github.com/nodesource/distributions/) as the version included with your distribution may be outdated.
+The extension is developed by members of the Swift Community and maintained by the [SSWG](https://www.swift.org/sswg/). The aim is to provide a first-class, feature complete extension to make developing Swift applications on all platforms a seamless experience.
 
-Next, clone this repository, and run the following commands:
+If you experience any issues or want to propose new features please [create an issue](https://github.com/swift-server/swift-vscode/issues/new) or post on the `#vscode-swift` channel on [Slack](https://swift-server.slack.com).
 
-```
-npm install
-npm run package
-```
+## Contributing
 
-This will generate a file with the **vsix** extension. To install this file, select **View ▸ Extensions** from the menu bar, click the triple dots, then select **Install from VSIX...**:
+The Swift for Visual Studio Code extension is a community driven project, developed by the amazing Swift community. Any kind of contribution is appreciated, including code, tests and documentation. For more details see [CONTRIBUTING.md](CONTRIBUTING.md).
 
-![](images/install-extension.png)
+## Installation
 
-Alternatively, if you just want to try this extension without installing it, open this project in Visual Studio Code and press **F5** to launch an instance of Visual Studio Code with the extension enabled.
+For the extension to work, you must have Swift installed on your system. Please see the [Getting Started Guide on Swift.org](https://www.swift.org/getting-started/) for details on how to install Swift on your system. Install the extension from [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=sswg.swift) and open a Swift package!
```

Comment:
Unfortunately, the official installation instructions for Linux and Windows are overly complicated and often don't work 😕

For example:

- The linked page doesn't actually explain how to install on Linux. You have to browse to the Download page for that, scroll down quite a bit, and follow lengthy steps, which are missing some steps that aren't obvious for beginners.
- The installation instructions for Windows may not work. I had to slightly adjust the commands to get them to work. That was fine for me, but it will trip up a lot of people. Also, `winget` seems to be the recommended way to install now, but it's still at version 5.5 (no patch releases).

All of this is why I started https://github.com/pwsacademy/swift-setup, but I assume we have to link to the official page here?

Reply:
Yeah I think for now we should stick with the official docs. If they don't work maybe we can get them updated cc @tomerd  

The new debs/rpms might change things again as well

Reply:
Yes native packages would be a big leap forward. I've been using the https://www.swiftlang.xyz packages for a while now, and they are much easier than use than the official releases.